scripts/generate_schemas.py

```diff
@@ -200,32 +204,63 @@ def __init__(self, version, schemaDir, outputFile):
         self.schemaDir = schemaDir
         self.outputFile = outputFile
         self.classes = []
-        for f in glob.glob(os.path.join(self.schemaDir, "*.avsc")):
-            self.classes.append(SchemaClass(f))
+        for avscFile in glob.glob(os.path.join(self.schemaDir, "*.avsc")):
+            self.classes.append(SchemaClass(avscFile))
+        self.requestClassNames = \
+            [clazz.name for clazz in self.classes
+                if re.search('Search.*Request', clazz.name)]
+        self.responseClassNames = \
+            [clazz.name for clazz in self.classes
+                if re.search('Search.*Response', clazz.name)]
+        self.postSignatures = []
+        for request, response in \
```

Comment:
I think we should prefer 
```python
 for request, response in zip(
        self.requestClassNames, self.responseClassNames):
```
here, and use line breaks as a last-resort. (See the section in pep8 on maximum line length) We'll definitely put this into the style guide.